packages/dashboard/src/components/pages/platforms/components/wrapper/wrapper.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```

Comment:
I was thinking we might need to move this wrapper outside of platforms folder, so that we can use it for other layouts as well. This way we don't need to have multiple wrapper components. What do you think?